### PR TITLE
fixed build badge

### LINKED ISSUE #136: Build badge on readme page points to the wrong repo
image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- fix #136 

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
@henrikstranneheim an easy review if you have time!

**Comment by northwestwitch** [comment]
aha, looks like the release of Flask2 has an impact on the tests of this one. I need to fix them then!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # cgbeacon2
 
-[![Build Status](https://travis-ci.org/Clinical-Genomics/MIP.svg?branch=master)](https://travis-ci.org/Clinical-Genomics/MIP)
+[![Build Status](https://travis-ci.com/Clinical-Genomics/cgbeacon2.svg?branch=master)](https://travis-ci.com/Clinical-Genomics/cgbeacon2)
```
Comment (henrikstranneheim) [inline README.md:3]: MIP badges that is interesting, 😄

**Review by henrikstranneheim — APPROVED** [review]
So far so good

**Pull request merged**